src/libs/actions/OnyxDerived/configs/reportAttributes.ts

```diff
@@ -73,9 +73,13 @@ export default createOnyxDerivedValueConfig({
                 dataToIterate = prepareReportKeys(updates);
                 recentlyUpdated = updates;
             } else if (!!transactionsUpdates || !!transactionViolationsUpdates) {
+                let transactionReportIds: string[] = [];
```

Comment:
```suggestion
                let transactionReportIDs: string[] = [];
```

Reply:
I'll update it with one of the following PRs, thanks